common/coat_of_arms/template_lists/color_lists.txt

```diff
@@ -255,6 +255,59 @@ color_lists = {
 			125 = "black"
 			250 = "black_empire_dark_red"
 		}
+		
+		###### Warcraft Culture Specific CoAs ######
+		
+		#### Humans ###
+		
+		special_selection = { 
+			trigger = { azerothian_coa_trigger = yes }
+			125 = "gold_alliance"
+			250 = "blue_alliance"
+		}
+		special_selection = {
+			trigger = { lordaeronian_coa_trigger = yes }
+			125 = "blue_alliance" #Blue Insignia
+			125 = "gold_alliance"
+			125 = "grey_dark"
+			250 = "silver_lordaeron"
+		}	
+		special_selection = {
+			trigger = { arathorian_coa_trigger = yes }
+			75  = "gold_alliance" #League of Arathor
+			125 = "silver_lordaeron"
+			250 = "red_strom"
+		}
+		special_selection = {
+			trigger = { alteraci_coa_trigger = yes }
+			125 = "brown_alterac"
+			125 = "black"
+			250 = "orange"
+		}
+		special_selection = {
+			trigger = { gilnean_coa_trigger = yes }
+			125 = "gold_alliance"
+			125 = "rose_gilneas"
+			125 = "grey_gilneas"
+			250 = "black"
+		}
+		special_selection = {
+			trigger = { tirassian_coa_trigger = yes }
+			125 = "gold_alliance"
+			125 = "blue_alliance" #Theramore
```

Comment:
Theramore may have its own colors. They have a separate culture.

Reply:
The triggers rely on each culture having a unique graphical culture coa gfx (i.e. azerothian_coa_gfx). Right now, Theramore shares its coa_gfx with Kul Tiras and also the disabled Pirate culture. One workaround is to define a different trigger condition such as the "western_" gfx but the trouble is that those are not exclusive to Theramore either. 

It is this same reason that I have disabled Hillsbrad, Dalaran, Pirate and Northern for now. 
![image](https://user-images.githubusercontent.com/77478662/149681273-f85756e4-a1a0-47ad-9e5d-48ef477b11fb.png)

I was actually planning to create a new issue to address cultures which are missing a unique coa_gfx. What do you think?

Reply:
Just add gfx for theramore.

Reply:
It worked!